src/test/java/net/liquidpineapple/pang/objects/TimeSystemTest.java

```diff
@@ -0,0 +1,73 @@
+package net.liquidpineapple.pang.objects;
+
+import net.liquidpineapple.pang.gui.NumberToken;
+import net.liquidpineapple.pang.gui.TimeSystem;
+import org.junit.Before;
+import org.junit.Test;
+import org.powermock.reflect.Whitebox;
+
+
+import java.sql.Time;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Erik on 13-9-2016.
+ */
+public class TimeSystemTest {
+
+  @Before
+  public void setUp() {
+    TimeSystem.getInstance();
+    TimeSystem.start();
+
+  }
+
+  @Test
+  public void TestsetFrozenTime() {
+    TimeSystem.getInstance().setFrozen(0);
+    assertEquals(0, TimeSystem.getFrozen());
+    TimeSystem.getInstance().setFrozen(10);
+    assertEquals(10, TimeSystem.getFrozen());
+  }
+
+  @Test
+  public void TeststopTime() {
+    TimeSystem.stop();
+    int oldTime = TimeSystem.getTime();
+    assertEquals(oldTime, TimeSystem.getTime());
```

Comment:
The problem is that i cant find an elegant solution for this, I already know a normal wait will just make Travis run other tests, which will fuck up the time because it's a singleton.